packages/swap/src/routes/swap.ts

```diff
@@ -110,6 +124,9 @@ router.get(
 
       swap = swapDepositChannel.swaps.at(0);
       failedSwap = swapDepositChannel.failedSwaps.at(0);
+      if (swapDepositChannel.affiliates.length > 0) {
+        affiliateBrokers = swapDepositChannel.affiliates;
```

Comment:
if we had `beneficiaries`, the `brokerCommissionBps` would be the sum of all commissions of al beneficiaries

Reply:
I don't think we should include it in that field. To me, it makes more sense to have the response match the request format. It might make sense to add a new field `totalBrokerCommissionBps`